Clockwise/VirtualClock.cs

```diff
@@ -84,7 +90,7 @@ private VirtualClock(DateTimeOffset? now = null, [CallerMemberName] string creat
 
             if (after == null || after <= now)
             {
-                scheduledTime = now.AddTicks(1);
```

Comment:
No idea why this was here in the first place to be honest.  It doesn't fit well with the "clock only advances when we tell it to, and scheduled actions are assumed to take `0` time.